packages/components/src/components/ErrorBox.tsx

```diff
@@ -7,10 +7,10 @@ const ShowError = styled.div`
   padding: 0.4em 0.8em;
 `;
 
-export const Error: React.FC<{ heading: string; children: React.ReactNode }> = ({
-  heading = "Error",
-  children,
-}) => {
+export const ErrorBox: React.FC<{
```

Comment:
Good name change

Reply:
The name changes are mainly motivated by conflicting with JS global functions (Error and eval) xD